lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -37,7 +37,15 @@ defmodule ExUnit.CaptureLog do
   Returns the binary which is the captured output.
 
   This function mutes the `:console` backend
-  and captures any log messages sent to Logger.
+  and captures any log messages sent to Logger from the calling
+  processes. It is possible to ensure explicit log messages from other
+  processes are captured by waiting for their exit or monitor
+  signal. This does not guarantee to capture VM generated log messages
+  when a process is spawned using a low level `Kernel` spawn function (e.g.
+  `Kernel.spawn/1`) raises an exception that isn't rescued or a thow that
+  isn't caught. A `Task`, orther OTP process, will send explicit logs that
+  are sent before its exit or monitor signal and will not cause VM generated
```

Comment:
Much clearer.  I found a couple more grammar/spelling issues.  I'm happy with it once those are addressed :).